Userland/Services/Taskbar/ShutdownDialog.cpp

```diff
@@ -18,24 +18,23 @@
 
 struct Option {
     DeprecatedString title;
-    Vector<char const*> cmd;
+    ShutdownDialog::Command command;
     bool enabled;
     bool default_action;
 };
 
-static Vector<Option> const options = {
-    { "Power off computer", { "/bin/shutdown", "--now", nullptr }, true, true },
-    { "Reboot", { "/bin/reboot", nullptr }, true, false },
-    { "Log out", { "/bin/logout", nullptr }, true, false },
+static Array const options = {
```

Comment:
If you keep `Array<Option>`, you don't have to specify the class in the initializer list every time.

Reply:
You can't do `Array<Option>` though, you have to do `Array<Option, 3>` which I find less appealing than specifying the class name.